geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/RedisSortedSetCommands.java

```diff
@@ -46,5 +46,7 @@
 
   byte[] zscore(RedisKey key, byte[] member);
 
+  List<byte[]> zpopmin(RedisKey key, int count);
```

Comment:
Gimme an alpha! Gimme a beta! What does that spell? Alphabeta, which is almost 'alpahbet'...

Reply:
Done